<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>581</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_set">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>380</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>581</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,3,1,3">
    <item row="0" column="0">
     <widget class="QLabel" name="m_lb_frame_shape">
      <property name="text">
       <string>Frame Shape</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="m_combo_shape">
      <item>
       <property name="text">
        <string>QFrame::NoFrame</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>QFrame::Box</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>QFrame::Panel</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>QFrame::StyledPanel</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>QFrame::HLine</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>QFrame::VLine</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>QFrame::WinPanel</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="m_frame_shadow">
      <property name="text">
       <string>Frame Shadow</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QComboBox" name="m_combo_shadow">
      <item>
       <property name="text">
        <string>QFrame::Plain</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>QFrame::Raised</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>QFrame::Sunken</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="m_lb_line_width">
      <property name="text">
       <string>Line Width</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="m_spb_line_width"/>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="m_lb_midline_width">
      <property name="text">
       <string>Midline Width</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="m_spb_midline_width"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
